src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,26 @@
+const jwtService = require('../services/jwt.service.js');
+
+const authMiddleware = (req, res) => {
```

Comment:
The `authMiddleware` function should include a `next` parameter to pass control to the next middleware function.